### ISSUE TITLE
tidytext data

### ISSUE THREAD

**Issue by EmilHvitfeldt** [EmilHvitfeldt/textdata#6]
- ~~nma_words~~
- ~~parts_of_speech~~
- [x] sentiments
    - ~~nrc~~
    - [x] bing
    - [x] loughran
    - [x] AFINN
- ~~stop_words~~
    - ~~onix~~
    - ~~SMART~~
    - ~~snowball~~

**Comment by EmilHvitfeldt** [comment]
@juliasilge now is the time to suggest more datasets if you want 😄I know there have been interest [earlier](https://github.com/juliasilge/tidytext/pull/47).

**Comment by juliasilge** [comment]
TBH I think NRC is going to have to be out-of-scope. I am still waiting on an email but the creator really does sound like he does not want this [data redistributed at all](http://saifmohammad.com/WebPages/lexicons.html#terms). 

I also think that stop words do not need to be in scope because of the excellent [stopwords](https://github.com/quanteda/stopwords) package, which tidytext depends on.

I am not sure the [parts of speech](https://archive.org/details/mobypartofspeech03203gut) dataset is worth spending time on because using this kind of unigram, tidy data approach performs quite poorly for POS tagging. You really do need a deep learning or otherwise more complex approach, such as that implemented in [cleanNLP](https://github.com/statsmaths/cleanNLP). I don't hear about anybody using this dataset really; I may just deprecate it, although I don't see a significant problem with the license either.

**Comment by EmilHvitfeldt** [comment]
Perfect, everything should be in order now.
